tests/file-force-hash-invalid/test.yaml

```diff
@@ -0,0 +1,17 @@
+requires:
+  min-version: 9
+
+command: |
+  ${SRCDIR}/src/suricata --set classification-file="${SRCDIR}/etc/classification.config" \
+      --set reference-config-file="${SRCDIR}/etc/reference.config" \
+      -l ${OUTPUT_DIR} -c "${TEST_DIR}/suricata.yaml" -T
```

Comment:
@inashivb is there no better way to have a SV test run `-T` ? just `pcap: false` ?

Reply:
I can't find it if there is.

Reply:
see `git grep '\-T'`